doozer

```diff
@@ -1051,8 +1155,7 @@ def cleanup(runtime):
 
     runtime.logger.info('Clearing out {}'.format(runtime.working_dir))
     if os.path.isdir(runtime.working_dir):
-        shutil.rmtree(runtime.working_dir, ignore_errors=True)
-        os.makedirs(runtime.working_dir)  # rmtree deletes the directory itself. recreate
+        exectools.cmd_assert('rm -rf {}/*'.format(runtime.working_dir))
```

Comment:
I know this already existed and you just re-wrote how it works, but since I'm just now seeing it, is there any assurance built in that this will never accidentally run somewhere like `/home/anybody` or just plain old `/`?

Reply:
No... and actually, this works on my laptop but not on buildvm. I'm thinking of just removing it entirely. I have no idea why it fails to do anything on buildvm, but it's not super useful anyways.